tests/reports/class-wc-tests-reports-orders-stats.php

```diff
@@ -865,7 +866,7 @@ public function test_populate_and_query_multiple_intervals() {
 						) { // As there are no tests for different timeframes, ignore these for now: $order_3_time, $order_4_time, $order_5_time, $order_6_time, $order_7_time
 							// One order with only 1 product.
 							$order = WC_Helper_Order::create_order( $customer->get_id(), $product );
-							$order->set_date_created( $order_time );
+							$order->set_date_created( $order_time + $iterations++ ); // offset each order by 1 second.
```

Comment:
Good catch. How about I just set the first order time to the beginning of the hour, that way there's no chance the last order will be in the next hour since we're only creating 144. (144 / 60 = 2.4 minutes)

Reply:
Fixed in e15755b.